packages/nextjs/app/builders/[address]/_components/builds/SubmitNewBuildBtn.tsx

```diff
@@ -0,0 +1,31 @@
+"use client";
+
+import { useRef } from "react";
+import { BuildFormModal } from "./BuildFormModal";
+import { PlusIcon } from "@heroicons/react/24/outline";
+import { useSubmitBuild } from "~~/hooks/useSubmitBuild";
+
+export const SubmitNewBuildBtn = () => {
```

Comment:
Let's call this `SubmitNewBuildButton`

Reply:
oh, maybe we can tweak the other `Btn` too? It's kind of a nit, but I don't see the benefit of using `Btn` (only that it has fewer letters?)

Reply:
Renamed at https://github.com/BuidlGuidl/SpeedRunEthereum-v2/pull/164/commits/c68f9b66dfb0ce82caf88acf3b3aef31e40fc4ec 🙌